core/src/main/java/com/linecorp/armeria/client/RequestOptions.java

```diff
@@ -89,6 +89,13 @@ default RequestOptionsBuilder toBuilder() {
      */
     long maxResponseLength();
 
+    /**
+     * Returns the amount of time to wait before aborting an {@link HttpRequest} when
+     * its corresponding {@link HttpResponse} is complete.
+     */
+    @Nullable
+    Long requestAutoAbortDelayMillis();
```

Comment:
Could you explain why this is `Long` rather than `long`? `responseAutoAbortDelayMillis()` is `long`.

Reply:
I had to use `Long` to check if it was set by the user or not.
Other options such as `responseTimeoutMillis` isn't allowed to use a negative value so we could check if it was set by checking the value:
https://github.com/line/armeria/pull/4913/files#diff-26ba9fc95f7dc4f9f751309a01bace23fad49f8be647b5a034178b9d3b2731deR313-R315

However, we allowed to use a negative value for `requestAutoAbortDelayMillis`, so I have to use `Long` if it was set or not.
https://github.com/line/armeria/pull/4913/files#diff-26ba9fc95f7dc4f9f751309a01bace23fad49f8be647b5a034178b9d3b2731deR308-R311

Reply:
Ahh, I see. Then could you add that to Javadoc, like we did in `maxResponseLength()` for disambiguation? e.g.

```
{@code null} disables this option and the request auto abort delay of a client will be used instead.
```

By the way, I find 'disables' here can sound confusing since it could mean that it will actually act as `-1`. Please consider using other verb instead of `disables` whereever applicable in this class. Or maybe just `null means the request auto abort delay of a client will be used instead.` ?

Reply:
> Or maybe just null means the request auto abort delay of a client will be used instead. ?

I like this sentence. Updated. Thanks!